docs/src/PageHeader.doc.js

```diff
@@ -0,0 +1,645 @@
+      title="Secondary action"
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+function SecondaryActionExample() {
+  const [open, setOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState([]);
+  const anchorRef = React.useRef(null);
+  const handleSelect = ({ item }) => {
+    if (selected.some((selectedItem) => selectedItem.value === item.value)) {
+      setSelected((selected) =>
+        selected.filter((selectedItem) => selectedItem.value != item.value),
+      );
+    } else {
+      setSelected((selected) => [...selected, item]);
+    }
+  };
+
+  return (
+    <Flex gap={8} direction="column" flex="grow">
+      <PageHeader
+        title="Product groups"
+        subtext="S. E. All products USD"
+        primaryAction={<Button color="red" text="Create product group" />}
+        secondaryAction={<Button text="Promote" />}
+      />
+      <Divider />
+      <PageHeader
+        title="Custom reports"
+        primaryAction={<Button color="red" text="Create new report" />}
+        secondaryAction={
+          <React.Fragment>
+            <Tooltip idealDirection="up" text="Board options">
+              <IconButton
+                accessibilityControls="page-header-example"
+                accessibilityHaspopup
+                accessibilityExpanded={open}
+                accessibilityLabel=""
+                icon="ellipsis"
+                iconColor="darkGray"
+                selected={open}
+                onClick={() => setOpen((prevVal) => !prevVal)}
+                ref={anchorRef}
+              />
+            </Tooltip>
+            {open && (
+              <Dropdown
+                id="page-header-example"
+                anchor={anchorRef.current}
+                onDismiss={() => {
+                  setOpen(false);
+                }}
+              >
+                <Dropdown.Item
+                  handleSelect={handleSelect}
+                  selected={selected}
+                  option={{
+                    value: 'Share Report',
+                    label: 'Share Report',
+                  }}
+                />
+                <Dropdown.Item
+                  handleSelect={handleSelect}
+                  selected={selected}
+                  option={{ value: 'Help center', label: 'Help center' }}
+                />
+                <Dropdown.Item
+                  handleSelect={handleSelect}
+                  selected={selected}
+                  badgeText="New"
+                  option={{
+                    value: 'Delete',
+                    label: 'Delete',
+                  }}
+                />
+              </Dropdown>
+            )}
+          </React.Fragment>
+        }
+      />
+    </Flex>
+  );
+}
+        `}
+      />
+    </MainSection.Subsection>
+
+    <MainSection.Subsection
+      description={`\`subtext\` should be used to add metadata about the content on the page, not to describe the page itself.`}
```

Comment:
This kinda reads more like a Do/Don't? It's fine here for now but a potential future improvement

Reply:
It's also a Don't